nav2_waypoint_follower/src/waypoint_follower.cpp

```diff
@@ -96,7 +96,8 @@ WaypointFollower::on_configure(const rclcpp_lifecycle::State & state)
     nav2_util::ServiceClient<robot_localization::srv::FromLL,
     std::shared_ptr<nav2_util::LifecycleNode>>>(
     "/fromLL",
-    node);
+    node,
+    true);
```

Comment:
```suggestion
    true /*creates and spins an internal executor*/);
```

Add this comment anywhere true / false are set (where false is 'Does not create'). I think this is a subtle but key detail that folks shouldn't overlook. This is not a software requirement but something that will help me in the future as I'm debugging user issues :-)